openapiart/bundler.py

```diff
@@ -598,6 +598,17 @@ def _required_restriction(self, schema_name, value):
                                 property=required, name=schema_name
                             )
                         )
+                    if "x-status" in value["properties"][required].keys():
```

Comment:
added this check as a required property should not have status deprecated, cause it then contradicts with the required nature it-self